### PR TITLE
[9.0] mgmtsystem

### PR DESCRIPTION
**Pull request by naousse**
module no more dependant from document_page

### CODE REVIEW

File: mgmtsystem/README.rst [hunk 1]
```diff
@@ -0,0 +1,107 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================================
+Management System -  Base
+================================
```
Comment (max3903) [inline mgmtsystem/README.rst]: Remove some = to match the text length

File: mgmtsystem/README.rst [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
+
+Changelog
+---------
+
+v9.0.1.0.0
+
+Here are the modification that have been done
```
Comment (dreispt) [inline mgmtsystem/README.rst]: Redundant line, please remove.

**Comment by dreispt** [comment]
After minor fixes to the README, I'm :+1:

PS: the rest of the text in the changelog section is also redundant IMO, because we have git version history for that.

File: mgmtsystem/README.rst [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
+
+Changelog
+---------
+
+v9.0.1.0.0
+
+* the module does no depends anymore on document_page module.
```
Comment (dreispt) [inline mgmtsystem/README.rst:74]: Good - this is relevant changelog information.

File: mgmtsystem/README.rst [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+Makes the Management System base Application available so that other Management System can be insalled
+
+Configuration
+=============
+
+No configuration required
+
+Usage
+=====
```
Comment (dreispt) [inline mgmtsystem/README.rst:28]: What do I expect in the Usage section? If I try it on runbot, a guide to where should I go and what should I do to check the module features.

File: mgmtsystem/__openerp__.py [hunk 5]
```diff
@@ -20,33 +20,22 @@
 ##############################################################################
 {
     "name": "Management System",
-    "version": "8.0.1.0.0",
+    "version": "9.0.1.0.0",
     "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
     "website": "http://www.savoirfairelinux.com",
     "license": "AGPL-3",
     "category": "Management System",
     "complexity": "normal",
```
Comment (dreispt) [inline mgmtsystem/__openerp__.py]: complexity is deprecated

File: mgmtsystem/models/mgmtsystem_system.py [hunk 6]
```diff
@@ -26,12 +26,11 @@ def own_company(self):
     return self.env.user.company_id.id
 
 
-class mgmtsystem_system(models.Model):
+class MgmtsystemSystem(models.Model):
 
     _name = 'mgmtsystem.system'
     _description = 'System'
 
     name = fields.Char(string='System', required=True)
```
Comment (dreispt) [inline mgmtsystem/models/mgmtsystem_system.py]: It would be nice to make it translatable

**Comment by dreispt** [comment]
- In the menuitems, use `group_mgmtsystem_viewer` instead of `base_group`
- The module adds a `system` Model, the a view for it is missing. I expect it under the Configuration menu.
- it would be nice to have a res.config page - please add that note to the Roadmap

File: mgmtsystem/README.rst [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+9.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Odoo SA <info@odoo.com>
```
Comment (max3903) [inline mgmtsystem/README.rst]: I don't think Odoo SA ever contributed to this module.

File: mgmtsystem/views/board_mgmtsystem_view.xml [hunk 8]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <menuitem
+            id="mgmtsystem_dashboard"
+            name="Reporting"
```
Comment (max3903) [inline mgmtsystem/views/board_mgmtsystem_view.xml]: name="Reports"

File: mgmtsystem/views/board_mgmtsystem_view.xml [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+            sequence="50"
+            parent="base.menu_reporting_dashboard"
+            action="open_board_mgmtsystem"/>
+    </data>
+</openerp>
```
Comment (max3903) [inline mgmtsystem/views/board_mgmtsystem_view.xml]: Remove this file.

File: mgmtsystem/views/menus.xml [hunk 10]
```diff
@@ -7,8 +7,6 @@
       name="Management System"
       sequence="150"
       groups="mgmtsystem.group_mgmtsystem_manager,base.group_user"
-      web_icon="images/mgmtsystem.png"
-      web_icon_hover="images/mgmtsystem-hover.png"
       />
 
     <menuitem
```
Comment (max3903) [inline mgmtsystem/views/menus.xml:10]: Add the "Reports" menu item in this file.

File: mgmtsystem/mgmtsystem_system.xml [hunk 11]
```diff
@@ -1,48 +0,0 @@
...
-    <menuitem
-      id="menu_mgmtsystem_configuration_system"
-      name="Systems"
-      parent="menu_mgmtsystem_configuration"
-      groups="group_mgmtsystem_manager"
-      sequence="60"
-      action="system_action"/>
-
-  </data>
-</openerp>
```
Comment (max3903) [inline mgmtsystem/mgmtsystem_system.xml:48]: This file should be renamed and moved, not deleted.

Comment (naousse) [inline mgmtsystem/mgmtsystem_system.xml:48]: mgmtsystem/mgmtsystem_system.xml is not deleted, i moved it to mgmtsystem_manual module

Comment (max3903) [inline mgmtsystem/mgmtsystem_system.xml:48]: @naousse Only the field `manual` should be moved to mgmtsystem_manual module. Everything else should stay in this module.

File: mgmtsystem/README.rst [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+     * nonconformities,
+     * immediate actions,
+     * preventive actions,
+     * corrective actions,
+     * improvement opportunities.
+
+Installation
+============
+
+Makes the Management System base Application available so that other Management System can be insalled
```
Comment (max3903) [inline mgmtsystem/README.rst:20]: can be installed.

File: mgmtsystem/README.rst [hunk 13]
```diff
@@ -0,0 +1,73 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Management System -  Base
```
Comment (max3903) [inline mgmtsystem/README.rst:6]: There is an extra space before Base.

**Comment by max3903** [comment]
@naousse Don't forget to include #110 improvements and to add the `Reporting` menu item,

File: mgmtsystem/__openerp__.py [hunk 14]
```diff
@@ -20,21 +20,11 @@
 ##############################################################################
 {
     "name": "Management System",
-    "version": "8.0.1.0.0",
+    "version": "1.0",
```
Comment (max3903) [inline mgmtsystem/__openerp__.py]: 9.0.1.0.0

**Comment by max3903** [comment]
:+1:

**Pull request merged**